src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -88,6 +88,30 @@ bool operator!=(const ApparentHorizon<Frame>& lhs,
 
 }  // namespace OptionHolders
 
+namespace OptionTags {
+template <typename InterpolationTargetTag, typename Frame>
+struct ApparentHorizon {
+  using type = OptionHolders::ApparentHorizon<Frame>;
+  static constexpr OptionString help{
+      "Options for interpolation onto apparent horizon."};
+  static std::string name() noexcept {
```

Comment:
@markscheel and I think that this can be simplified by just using the string obtained from a `get_name` or similar from `PrettyType.hpp` on the `InterpolationTargetTag` template parameter. Then, the `InterpolationTargetTag`s don't need to also implement a `name()` function.

Reply:
Looks like `pretty_type::short_name<InterpolationTargetTag>()` will do it.  I will make this change.  This way there is one less thing that needs to be in each InterpolationTargetTag, and less chances for bugs (e.g. having `AhA::name()` return "AhB").

Reply:
Use `option_name<...>()`. It defaults to the pretty_type but can be overridden by providing a `name`.